src/devices/vdml_gps.c

```diff
@@ -81,7 +81,7 @@ double gps_get_error(uint8_t port) {
 	return_port(port - 1, rtv);
 }
 
-gps_status_s_t gps_get_status(uint8_t port) {
+gps_status_s_t gps_get_position_and_orientation(uint8_t port) {
 	gps_status_s_t rtv = GPS_STATUS_ERR_INIT;
 	if (!claim_port_try(port - 1, E_DEVICE_GPS)) {
 		return rtv;
```

Comment:
Another alternative is typecasting the input parameter (data) from gps_status_s_t or whatever it is to the VEX API struct.

Reply:
Outside of the scope of this PR but the IMU API may have the same issue.

Reply:
This function currently converts the VEX API struct (V5_DeviceGpsAttitude) into gps_status_s_t, and returns the gps_status_s_t. If we want to cast gps_status_s_t into V5_DeviceGpsAttitude, should the return type be just V5_DeviceGpsAttitude then?